events/wmt21.md

```diff
@@ -44,165 +44,166 @@ Wednesday, November 10, 2021
-| 10:30 – 12:00 | _**MiSS@WMT21: Contrastive Learning-reinforced Domain Adaptation in Neural Machine Translation**_  <br>Zuchao Li, Masao Utiyama, Eiichiro Sumita and Hai Zhao |
-| 10:30 – 12:00 | _**The Fujitsu DMATH Submissions for WMT21 News Translation and Biomedical Translation Tasks**_  <br>Ander Martinez |
-| 10:30 – 12:00 | _**Adam Mickiewicz University’s English-Hausa Submissions to the WMT 2021 News Translation Task**_  <br>Artur Nowakowski and Tomasz Dwojak |
-| 10:30 – 12:00 | _**eTranslation’s Submissions to the WMT 2021 News Translation Task**_  <br>Csaba Oravecz, Katina Bontcheva, David Kolovratník, Bhavani Bhaskar, Michael Jellinghaus and Andreas Eisele |
-| 10:30 – 12:00 | _**The University of Edinburgh’s Bengali-Hindi Submissions to the WMT21 News Translation Task**_  <br>Proyag Pal, Alham Fikri Aji, Pinzhen Chen and Sukanta Sen |
-| 10:30 – 12:00 | _**The Volctrans GLAT System: Non-autoregressive Translation Meets WMT21**_  <br>Lihua Qian, Yi Zhou, Zaixiang Zheng, Yaoming ZHU, Zehui Lin, Jiangtao Feng, Shanbo Cheng, Lei Li, Mingxuan Wang and Hao Zhou |
-| 10:30 – 12:00 | _**NVIDIA NeMo’s Neural Machine Translation Systems for English-German and English-Russian News and Biomedical Tasks at WMT21**_  <br>Sandeep Subramanian, Oleksii Hrinchuk, Virginia Adams and Oleksii Kuchaiev |
-| 10:30 – 12:00 | _**Facebook AI’s WMT21 News Translation Task Submission**_  <br>Chau Tran, Shruti Bhosale, James Cross, Philipp Koehn, Sergey Edunov and Angela Fan |
-| 10:30 – 12:00 | _**Tencent Translation System for the WMT21 News Translation Task**_  <br>Longyue Wang, Mu Li, Fangxu Liu, Shuming Shi, Zhaopeng Tu, Xing Wang, Shuangzhi Wu, Jiali Zeng and Wen Zhang |
-| 10:30 – 12:00 | _**HW-TSC’s Participation in the WMT 2021 News Translation Shared Task**_  <br>Daimeng Wei, Zongyao Li, Zhanglin Wu, Zhengzhe Yu, Xiaoyu Chen, Hengchao Shang, Jiaxin Guo, Minghan Wang, Lizhi Lei, Min Zhang, Hao Yang and Ying Qin |
-| 10:30 – 12:00 | _**LISN @ WMT 2021**_  <br>Jitao Xu, Minh Quang Pham, Sadaf Abdul Rauf and François Yvon |
-| 10:30 – 12:00 | _**WeChat Neural Machine Translation Systems for WMT21**_  <br>Xianfeng Zeng, Yijin Liu, Ernan Li, Qiu Ran, Fandong Meng, Peng Li, Jinan Xu and Jie Zhou |
-| 10:30 – 12:00 | _**Small Model and In-Domain Data Are All You Need**_  <br>Hui Zeng |
-| 10:30 – 12:00 | _**The Mininglamp Machine Translation System for WMT21**_  <br>Shiyu Zhao, Xiaopu Li, Minghui Wu and Jie Hao |
-| 10:30 – 12:00 | _**The NiuTrans Machine Translation Systems for WMT21**_  <br>Shuhan Zhou, Tao Zhou, Binghao Wei, Yingfeng Luo, Yongyu Mu, Zefan Zhou, Chenglong Wang, Xuanjun Zhou, Chuanhao Lv, Yi Jing, Laohu Wang, Jingnan Zhang, Canan Huang, Zhongxiang Yan, Chi Hu, Bei Li, Tong Xiao and Jingbo Zhu |
+| 10:30 – 12:00 | _**[GTCOM Neural Machine Translation Systems for WMT21](https://www.statmt.org/wmt21/pdf/2021.wmt-1.3.pdf)**_  <br>Chao Bei and Hao Zong |
```

Comment:
It'll make life a bit easier to put the formatting *inside* the [...] and to use just `***` instead of mixing `*` and `_` which are equivalent.